packages/func/src/executors/start/schema.d.ts

```diff
@@ -1,3 +1,17 @@
 export interface StartExecutorSchema {
   port: number;
+  cors: string;
+  corsCredentials: boolean;
+  timeout: number;
+  useHttps: boolean;
+  cert: string;
+  password: string;
+  languageWorker: string;
+  noBuild: boolean;
+  enableAuth: boolean;
+  functions: string;
+  verbose: boolean;
+  dotnetIsolatedDebug: boolean;
+  enableJsonOutput: boolean;
+  jsonOutputFile: string;
```

Comment:
It seems that none of these are actually needed to run the functions.
In that case, I would suggest to add all of those as optional properties and remove the default value in the schema.json.

BUT, if we go even further. Is there an actual reason to have all of those typed here? It feels to me that it would be much more beneficial (and future proof) to include an "additional flags" that will be just passed as to the command.
This way, you can pass exactly what  you need and there won't be a need to maintain this list.

Reply:
That's a better idea and more maintainable.